tests/test_erosita.py

```diff
@@ -141,29 +141,29 @@ def test_filter_on_rect_region(self):
     def test_filter_on_positions_one_pos(self):
         # Testing for one RA and DEC input as a list
         self.defaults.filter_on_positions([129.55, 1.50])
-        self.assertEqual(self.defaults.filtered_obs_ids, ['300007'])
+        assert_array_equal(self.defaults.filtered_obs_ids, np.array(['300007', '300008']))
```

Comment:
the fov of erosita was updated to be larger, so some filters now pick up another observation, so I had to update the expected result of my tests